### PR TITLE
ci: split up e2etests, dont log so aggresively

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

File: src/client/client_api/transfers/balance_management.rs [hunk 1]
```diff
@@ -276,7 +276,7 @@ mod tests {
             let _ =
                 retry_loop_for_pattern!( client.get_balance(), Ok(bal) if *bal == correct_balance);
 
-            if count == 50 {
+            if count == 7 {
```
Comment (connec) [inline src/client/client_api/transfers/balance_management.rs:279]: How did you choose 7, out of curiosity?

**Review by connec — APPROVED** [review]
Only issue I see with this would be new test modules not being picked up automatically, but not a big deal.

Would it make sense, as a further future enhancement, to run the different modules in parallel, or would that not work against a single network?

File: src/client/client_api/transfers/balance_management.rs (hunk 1 above)
Comment (joshuef) [reply src/client/client_api/transfers/balance_management.rs:279]: hmm, i think this is what we were doing before.

but it's a random number. Everything is in seq, 7 in a row means stuff is going okay. 50 in a row takes like 5 mins on my machine, so probs adds 15 mins to CI

**Review by connec — APPROVED** [review]

**Comment by joshuef** [comment]
> Would it make sense, as a further future enhancement, to run the different modules in parallel, or would that not work against a single network?

Im not sure we can have one network used by many jobs. But it would make sense if we could. They _might_ slow each other down though.

Down the line having a constant DO testnet to test against may be the answer here (when not having breaking changes...)

**Pull request merged**